src/category_theory/monoidal/Mon_.lean

```diff
@@ -282,5 +483,4 @@ Projects:
   (We've already got `Mon_ (Module R) ≌ Algebra R`, in `category_theory.monoidal.internal.Module`.)
 * Can you transport this monoidal structure to `Ring` or `Algebra R`?
   How does it compare to the "native" one?
-* Show that if `C` is braided then `Mon_ C` is naturally monoidal.
```

Comment:
In the interest of giving people things to do, perhaps we can add a further suggestion here:
```
* Show that when `C` is braided, the forgetful functor `Mon_ C \func C` is monoidal.
```
Feel free to add anything else here, too; I think this sort of comment is helpful for the growth of the library.